pyrato/rap.py

```diff
@@ -1,4 +1,6 @@
-"""Room acoustic parameters
+"""
+Sub-module implementing calculations of room acoustic parameters from
+simulated or experimental data.
```

Comment:
data or (room) impulse responses?

Reply:
Currently it's based on EDCs only, but for others it might be RIRs as well. So It think data is fitting here.